src/runtime/runtime_internal.h

```diff
@@ -209,12 +209,6 @@ ALWAYS_INLINE T min(const T &a, const T &b) {
     return a < b ? a : b;
 }
 
-template<typename T, typename U>
-ALWAYS_INLINE T reinterpret(const U &x) {
-    T ret;
```

Comment:
I think a better fix might be to just initialize `T ret = T()`.

Reply:
That's definitely better than the status quo, but I stand by my assertion that this function is a bad idea, requiring too much care on the part of the caller.